javascript/builtins/String.json

```
@@ -1824,9 +1824,16 @@
               "firefox_android": {
                 "version_added": "4"
               },
-              "ie": {
-                "version_added": "4"
-              },
+              "ie": [
+                {
+                  "version_added": "5.5"
+                },
+                {
+                  "version_added": "4",
+                  "partial_implementation": true,
+                  "notes": "No support for a function as a second parameter before version 5.5."
```

This rewords the note to avoid restating the version number (and to use a style more like other notes in the repo).

```suggestion
                  "notes": "A replacement function as second argument is not supported."
```

---

I used other notes sections in this file for inspiration and they do restate version numbers :-) But I agree, your wording is much better. Fixed.